export/build-requirements.txt

```diff
@@ -0,0 +1,3 @@
+pexpect==4.9.0 --hash=sha256:9eaf9cf5e3332373fab8184f455314ac01ebbb33e110331bc9e8e12daea1f68e
+ptyprocess==0.7.0 --hash=sha256:b6194d9cb391fd7e02697548610c8eb18ba0226af2a9584bda50605958dc1a6b
+
```

Comment:
This extra newline is causing the diff test to fail.

Reply:
womp womp. thanks for the catch, will fix

Reply:
fixed in a55a706b (though There Will Be Squashes)